lib/improver/spotdata/common_functions.py

```diff
@@ -482,3 +494,40 @@ def extract_ad_at_time(additional_diagnostics, time, time_extract):
         cubes = additional_diagnostics[key]
         ad_extracted[key] = extract_cube_at_time(cubes, time, time_extract)
     return ad_extracted
+
+
+def iris_time_to_datetime(time):
+    """
+    Convert iris time to pythond datetime object.
+
+    Args:
+    -----
+    time : iris.coord.Coord
+        Iris time coordinate element(s).
+
+    Returns:
+    --------
+    list of datetime.datetime objects
+        The time element(s) recast as a python datetime object.
+
+    """
+    time.convert_units('seconds since 1970-01-01 00:00:00')
+    return [dt.utcfromtimestamp(value) for value in time.points]
+
+
+def dt_to_utc_hours(dt_in):
+    """
+    Convert python datetime.datetime into hours since 1970-01-01 00Z.
+
+    Args:
+    -----
+    dt_in : datetime.datetime object
+
+    Returns:
+    --------
+    hours since epoch : float
+
+    """
+    from time import mktime
```

Comment:
We haven't been importing into functions elsewhere, so I think that this import would be moved to the top of this module.